Makefile

```diff
@@ -73,6 +74,23 @@ ldflags := $(strip $(ldflags))
 
 BUILD_FLAGS := -tags "$(build_tags_comma_sep)" -ldflags '$(ldflags)' -trimpath
 
+build-reproducible-amd64: go.sum $(BUILDDIR)/
```

Comment:
`make: *** No rule to make target `wasmd/build', needed by `build-reproducible-amd64'.  Stop.`